lib/src/command.dart

```diff
@@ -10,6 +10,17 @@ import 'global_packages.dart';
 import 'log.dart' as log;
 import 'system_cache.dart';
 
+/// All of the aliases used for [PubCommand] subclasses.
+///
+/// Centralized so invocations with aliases can be normalized to the primary
+/// command name when sending telemetry.
+const pubCommandAliases = {
```

Comment:
Nit: this table could have had the other direction:
```
cons pubCommandAliases = {
  'dependencies': 'deps',
  'tab': 'deps',
  
}
```

Reply:
True. Since we need to map both ways, I figured I'd keep the existing structure so it's easy to eyeball the mapping from command -> alias